packages/core/cache/nuxt/index.js

```diff
@@ -1,93 +1,132 @@
 /* eslint-disable */
-import path from 'path';
-import { createMiddleware } from '@vue-storefront/core/server';
+import { isAbsolute, resolve } from 'path';
 import { Logger } from '@vue-storefront/core';
 
-function rendererFactory (renderFn) {
-  const renderer = this.nuxt.renderer;
-  const renderRoute = renderer.renderRoute.bind(renderer);
+/**
+ * Helper function that imports default handler by package name
+ */
+function requirePackage (name) {
+  const path = isAbsolute(name) || name.startsWith('.')
+    ? resolve(process.cwd(), name)
+    : require.resolve(name, { paths: [ process.cwd() ] });
 
-  renderer.renderRoute = (route, context) => {
-    const render = () => renderRoute(route, context)
-    return renderFn(route, context, render)
-  }
+  return require(path).default;
 }
 
-const createInvalidators = (options) => {
-  const invalidatorsRaw = options.server.invalidators || []
+/**
+ * Adds endpoint to invalidate cache
+ */
+function createInvalidationEndpoint (driver, options) {
+  if (!options || !options.endpoint || !options.invalidators) {
+    return;
+  }
 
-  return invalidatorsRaw.map(invalidator => {
-    if (typeof invalidator === 'string') {
-      return require(path.resolve(invalidator)).default;
+  const handler = async (request, response) => {
+    try {
+      // Resolve invalidator paths to packages
+      const invalidators = options.invalidators.map(invalidator => {
```

Comment:
I think `driver.invalidate` is an invalidator. These methods just prepares list of tags to invalidate. Maybe we should use a different name to miss confusing it with `driver's invalidators`? Let me know wdyt

Reply:
I renamed them to `handlers` so in config looks like this:

```javascript
['@vue-storefront/cache/nuxt', {
  invalidation: {
    endpoint: '/cache-invalidate',
    key: 'uniqueKey',
    handlers: []
  },
  driver: []
}]
```